src/index.html

```diff
@@ -4,9 +4,469 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="MOYO companys logo image"
+        >
+      </a>
+
+      <nav class="header__nav">
```

Comment:
its better to make nav as block, so we can reuse it (on my opinion)

Reply:
In commit "Added styles for block "Header" and passed all tests for header" I do the navigation as element, because we didn't reuse it.

Reply:
@KuharchukDmytro https://github.com/mate-academy/layout_catalog/pull/2639/commits/c253da77025af2c5bf8a48c69698108566818f7b